resources/packs/azure/azure.lr

```diff
@@ -572,7 +572,7 @@ private azure.subscription.storageService.account.dataProtection {
 }
 
 // Azure Storage Account Blob Service Properties
-private azure.subscription.storageService.account.blobService.properties {
+private azure.subscription.storageService.account.blobService.properties @defaults("id hourMetrics minuteMetrics") {
```

Comment:
@chris-rock I think this might be too much for the defaults. Maybe we should just use the `id` 🤔 ...

<img width="1440" alt="image" src="https://github.com/mondoohq/cnquery/assets/49754039/b99ec35e-5bc6-4182-bf5a-1b9a2e05595a">

```suggestion
private azure.subscription.storageService.account.blobService.properties @defaults("id") {
```